ckanext/ontario_theme/templates/package/resource_read.html

```diff
@@ -0,0 +1,43 @@
+{% ckan_extends %}
+
+{% block resource_actions_inner %}
+  {% if h.check_access('package_update', {'id':pkg.id }) %}
+    <li>{% link_for _('Edit'), named_route='resource.edit', id=pkg.name, resource_id=res.id, class_='btn btn-secondary', icon='wrench' %}</li>
```

Comment:
This seems to be the master branch template, not 2.8.2. There is a chance this will break with our ckan version. Not all aspects have been migrated to flask yet.